### PR TITLE
Prepare for distutils.version being removed in Python 3.12

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Prepare for distutils.version being removed in Python 3.12

Relates to https://github.com/ansible-collections/overview/issues/45#issuecomment-999911221

##### ISSUE TYPE

- Bugfix Pull Request

### CODE REVIEW

File: plugins/module_utils/version.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Felix Fontein <felix@fontein.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""Provide Version object to compare version numbers."""
```
Comment (felixfontein) [inline plugins/module_utils/version.py]:
```suggestion
"""Provide version object to compare version numbers."""
```

**Review by felixfontein — APPROVED** [review]

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing!

**Comment by felixfontein** [comment]
Please consider creating a follow-up PR based on https://github.com/ansible-collections/community.general/pull/3984.

**Comment by felixfontein** [comment]
Please consider creating a follow-up PR based on https://github.com/ansible-collections/community.general/pull/3984.

**Comment by Andersson007** [comment]
@felixfontein yep, will do, thanks!

**Pull request merged**